internal/connection/connection.go

```
@@ -50,8 +34,31 @@
+		packet = RawNPSPacket{}
+	case 8228:
+		log.Println("Handling packet for port 8228")
+		packet = RawNPSPacket{}
+	case 7003:
+		log.Println("Handling packet for port 7003")
+		packet = RawNPSPacket{}
+	case 43300:
+		log.Println("Handling packet for port 43300")
+		packet = RawNPSPacket{}
+	default:
+		log.Panicln("Unknown port: %d", conn.LocalAddr().(*net.TCPAddr).Port)
```

golangci-lint (1.62.2)

printf: log.Panicln call has possible Printf formatting directive %d

(govet)